assets/templates/dataset-filter/list-selector.tmpl

```diff
@@ -22,12 +22,12 @@
 		</div>
 	</div>
 </div>
-<div class="adjust-font-size--16">
+<div class="adjust-font-size--18 line-height--32">
 	<div class="page-content link-adjust">
 		<div class="wrapper">
 			<div class="col-wrap">
 				<div class="col">
-					<form id="filter-form" class="form adjust-font-size--16 clear-left" method="post" action="{{.Data.RangeData.URL}}">
+					<form id="filter-form" class="form adjust-font-size--18 line-height--32 clear-left" method="post" action="{{.Data.RangeData.URL}}">
```

Comment:
You can remove the `adjust-font-size--18` here as you have set it on line 25 above, as well as `line-height--32` as it isn't making a difference here